Doc/library/gzip.rst

```diff
@@ -234,9 +236,26 @@ Command line options
 
    If *file* is not specified, read from :attr:`sys.stdin`.
 
+.. cmdoption:: --fast
+
+   Indicates the fastest compression method (less compression).
+
+   .. code-block:: shell-session
+
+      $ python -m gzip --fast file
+
+.. cmdoption:: --best
+
+   Indicates the slowest compression method (best compression).
+   This is the default method if you do not specify any flag.
```

Comment:
Out of date?

Reply:
@vadmium sure, if we use the level 6, it's not the `--best` flag.

thanks

Reply:
sure, thanks